Lib/configparser.py

```diff
@@ -987,7 +987,10 @@ def _read(self, fp, fpname):
             cur_indent_level = first_nonspace.start() if first_nonspace else 0
             if (cursect is not None and optname and
                 cur_indent_level > indent_level):
-                cursect[optname].append(value)
+                if cursect[optname]:
+                    cursect[optname].append(value)
+                else:
+                    raise ParsingError(f"No-value option {optname} cannot be continued")
```

Comment:
This is an invalid invocation of `ParsingError`. The sole argument of `ParsingError` is the source filename. See how `_handle_error()` constructs `ParsingError` instances.

More importantly, instead of instantiating `ParsingError`, make a subclass like `MultilineContinuationError`. That way it's obvious what went wrong without having to parse the message string.

Also, unrelatedly, I would make the condition here in line 990 explicitly check for `is None`.

Reply:
Okay, sure.
what do you think of this 
```
configparser.MultilineContinuationError: File contains a key without value if unintended scenario, please don't add any extra space within the section.
file: 'test', line: 3
' KEY2 = VAL2\n'
```